deployments/icesat2/config/staging.yaml

```diff
@@ -17,6 +17,32 @@ pangeo:
       extraEnv:
         OAUTH_CALLBACK_URL: "https://staging.aws-uswest2.pangeo.io/hub/oauth_callback"
     singleuser:
+      profileList:
+        - display_name: "Staging Pangeo-notebook"
+          description: "https://github.com/pangeo-data/pangeo-docker-images/tree/master/pangeo-notebook"
+          kubespawner_override:
+            image: pangeo/pangeo-notebook:master
```

Comment:
pointing to 'master' tag on staging hub allows testing staging images before tagging a 'production' release on pangeo-docker-images. So this is setup is very much coupled to pangeo-docker-images where production releases are manually tagged with calver data and 'latest' after confirming they work